dist/t/spec/spec_helper.rb

```diff
@@ -11,8 +11,33 @@
   config.disable_monkey_patching!
 
   # Run specs in random order to surface order dependencies
-  config.order = :random
+  ##config.order = :random  # we will follow one workflow for now...
```

Comment:
Because this is one big workflow test against a live instance without any fixtures/factories...